src/main/java/com/entropyteam/entropay/employees/controllers/ContractController.java

```diff
@@ -14,10 +13,20 @@
 @CrossOrigin
 @RequestMapping(value = "/contracts", produces = MediaType.APPLICATION_JSON_VALUE)
 public class ContractController extends BaseController<ContractDto, UUID> {
-
+    private final ContractService contractService;
     @Autowired
     public ContractController(ContractService contractService) {
         super(contractService);
+        this.contractService = contractService;
+    }
+
+    @PostMapping("/v2")
+    public ResponseEntity<ContractDto> create(@RequestBody ContractDto contract) {
+        return ResponseEntity.ok(contractService.create(contract));
     }
```

Comment:
This method could be deleted because the endpoint POST /contracts in the baseController will be processed by `contractService.create(contract)`